snippets/saved_replies.md

```diff
@@ -0,0 +1,103 @@
+
+`Signed-Off-By: Your Name <your@email.address>`
+
+If you are using GitHub through the web interface, it's quickest to close this PR and open a new one with the appropriate line.
+
+If you are using Git on the command line, it is probably quickest to amend and force push. You can do that with
+````
+git commit --amend --signoff
+git push -f $remote $remote_branch_for_pr
+````
+
+As always, be careful when force-pushing.
+
+
+## Moratorium on integrations (SD & Alertmanager)
+
+Thanks for suggesting this change. Unfortunately, we are currently not accepting new integrations.
+
+We can only provide the stability and performance we want to provide if we can test integrations in an automated and scalable fashion. For this reason, we are suggesting people integrate with the help of our generic interfaces by either integrating natively or putting a rewriting proxy in the middle. We would be more than happy to list this on our integrations page.
+
+Even if existing integrations can not be tested in an automated fashion, we will not remove them for reasons of compability. This also means that any additions we take on, or any changes to existing integrations we make or accept, will mean maintaining and testing these until at least the next major version, realistically even beyond that.
+
+You are free to question this answer on our [developer mailing list](https://groups.google.com/forum/#!forum/prometheus-developers), but it's unlikely that you will get a different answer.
+
+
+# Issue or PR
+
+
+## Duplicate
+
+Thanks for reporting this. It seems we already have this issue tracked at XXX
+
+Thus, we are closing this one for now. If you think this was in error, don't hesitate to comment.
+
+
+## Non-trivial
+
+Thanks for this. This topic is not as easy as it looks; we actually already had some discussions and considerations about this topic, for more context please look here XXX
+
+
+## Out of scope
+
+Thank you for taking the time to write this out. Unfortunately, this is out of scope for what we are trying to achieve. You can find more information regarding our reasoning here XXX
+
+
+#### Philosophy
+
+As this goes against our core philosophy, it's near certain that Prometheus will never address this.
+
+You might want to look at an alternative, such as XXX
+
+
+#### Non-Goal
+
+This is actually one of our non-goals. we are unlikely to change our minds on this, but feel free to raise this issue on our [developer mailing list](https://groups.google.com/forum/#!forum/prometheus-developers). Please understand if the answer still remains a simple "no" pointing to our docs.
+
+
+#### Against goal / design decision
+
+As this goes against one of our stated goals/design decisions, we are unlikely to change our minds on this, but feel free to raise this issue on our [developer mailing list](https://groups.google.com/forum/#!forum/prometheus-developers). Please understand if the answer still remains a simple "no" pointing to our docs.
```

Comment:
This seems exactly the same as a non-goal. I think what you were going for is fundamental incompatibility with design decisions. In cases like that I like to say something along the lines of:

"Thank you for taking the time to open this issue/pull request! We understand your need for this, however, it is fundamentally incompatible with the design of Prometheus. If you are interested in the details, we're more than happy to discuss this on the [developer mailing list](https://groups.google.com/forum/#!forum/prometheus-developers). :slightly_smiling_face: "

Reply:
This is reference to https://github.com/prometheus/docs/issues/1038#issuecomment-394067089 - it would point to different URLs, and a goal will change a lot less often than a design decision. Also, please re-read in the context of both blurbs spliced together.

Reply:
Same as above re last sentence.